app/app.rb

```diff
@@ -23,9 +23,16 @@ class Makersbnb < Sinatra::Base
                           address: params[:address],
                           description: params[:description],
                           price: params[:price])
-    redirect '/spaces' # once it exists redirect to specific space page
+    redirect '/spaces'
   end
 
+  get '/spaces/:id' do
+    space = Space.all(id: params[:id])
```

Comment:
Should only ever retrieve one space, so should be '@space = Space.all(id: params[:id])' and then pass the single space to the erb.